src/Components/v2/ArtistBio.tsx

```diff
@@ -16,20 +15,15 @@ export const MAX_CHARS = {
 }
 
 export class ArtistBio extends React.Component<ArtistBioProps> {
-  renderReadMore = size => {
-    return (
-      <ReadMore
-        onReadMoreClicked={this.props.onReadMoreClicked}
-        maxChars={size === "xs" ? MAX_CHARS.xs : MAX_CHARS.default}
-        content={this.props.bio.biography_blurb.text}
-      />
-    )
-  }
   render() {
+    const { bio } = this.props
     return (
       <Serif size="3">
-        <Media at="xs">{this.renderReadMore("xs")}</Media>
-        <Media greaterThan="xs">{this.renderReadMore("rest")}</Media>
+        <span
+          dangerouslySetInnerHTML={{
```

Comment:
@zephraph copied this from Palette's `ReadMore`, have no clue what it does, let me know if you have any concerns with this.

Reply:
I talked to @sweir27 about this. We should probably follow up later to make sure we're covering the XSS cases.

Reply:
I think this is about accepting markdown as input (from Artsy admins or partners), and then generating and rendering the HTML (with links, etc.).

Sanitizing the generated HTML is probably well worth a follow up, likely in https://github.com/artsy/metaphysics/blob/ac9c2bbadf84b00ad3bcb5bf271039f7909b334f/src/schema/v1/fields/markdown.ts